jkube-kit/jkube-kit-quarkus/src/main/java/org/eclipse/jkube/quarkus/generator/QuarkusGenerator.java

```diff
@@ -145,7 +145,7 @@ private AssemblyConfiguration createAssemblyConfiguration(String targetDir, Asse
 
   private AssemblyFileSet getJvmFilesToInclude() {
     AssemblyFileSet.AssemblyFileSetBuilder fileSetBuilder =
-        getFileSetWithFileFromBuildThatEndsWith("-runner.jar");
+        getFileSetWithFileFromBuildThatEndsWith("-run.jar");
```

Comment:
Umm, I think you're trying to modify QuarkusGenerator to add support for fast jar(https://github.com/eclipse/jkube/issues/632) . In my opinion, this change should go in a separate PR

Reply:
I opened the https://github.com/eclipse/jkube/pull/638